de-lib/src/main/java/org/iplantc/de/client/models/userSettings/UserSetting.java

```diff
@@ -0,0 +1,53 @@
+package org.iplantc.de.client.models.userSettings;
+
+import org.iplantc.de.client.models.diskResources.Folder;
+
+/**
+ * @author aramsey
+ */
+public interface UserSetting {
+
+    Boolean isEnableAnalysisEmailNotification();
+    void setEnableAnalysisEmailNotification(Boolean enable);
+
+    Boolean isEnableImportEmailNotification();
+    void setEnableImportEmailNotification(Boolean enable);
+
+    String getDefaultFileSelectorPath();
+	void setDefaultFileSelectorPath(String value);
```

Comment:
I like how some of these getters and setters are aligned with each other, but this one and a few others are a little off (the 2 above are not aligned this way either, but they are still indented normally).